shell/platform/embedder/embedder.h

```diff
@@ -1215,6 +1262,31 @@ typedef struct {
   FlutterPlatformViewIdentifier platform_view_id;
   /// A textual tooltip attached to the node.
   const char* tooltip;
+  // The number of string attributes associated with the `label`.
+  size_t label_attribute_count;
+  // Array of string attributes associated with the `label`.
+  // Has length `label_attribute_count`.
+  const FlutterStringAttribute** label_attributes;
+  // The number of string attributes associated with the `hint`.
+  size_t hint_attribute_count;
+  // Array of string attributes associated with the `hint`.
+  // Has length `hint_attribute_count`.
+  const FlutterStringAttribute** hint_attributes;
```

Comment:
Not a nit, but just out of curiosity do iOS/Android support all these today? This seems fine either way since I'd imagine they _probably_ should -- or at least value and label and their increased/decreased versions.

Reply:
Yup:

1. iOS: [here](https://github.com/flutter/engine/blob/34295287fd3b147cb52f92fc5b1cf99a1b811ea1/shell/platform/darwin/ios/framework/Source/SemanticsObject.mm#L612-L618) and [here](https://github.com/flutter/engine/blob/34295287fd3b147cb52f92fc5b1cf99a1b811ea1/shell/platform/darwin/ios/framework/Source/SemanticsObject.mm#L440-L447)
2. Android: [here](https://github.com/flutter/engine/blob/9fae689d10a9fc7615b9b185172b1bed5c8d40a1/shell/platform/android/io/flutter/view/AccessibilityBridge.java#L2549-L2569), [here](https://github.com/flutter/engine/blob/9fae689d10a9fc7615b9b185172b1bed5c8d40a1/shell/platform/android/io/flutter/view/AccessibilityBridge.java#L2632-L2672), [here](https://github.com/flutter/engine/blob/9fae689d10a9fc7615b9b185172b1bed5c8d40a1/shell/platform/android/io/flutter/view/AccessibilityBridge.java#L2294-L2315)